tests/test_admin/test_view.py

```diff
@@ -1575,10 +1575,10 @@ def test_post_link_and_enroll_into_program(
         expected_messages = {
             (messages.SUCCESS, "2 new learners were added to {}.".format(self.enterprise_customer.name)),
             (messages.WARNING, pending_user_message.format(unknown_email))
-        } | set([
+        } | {
```

Comment:
?????

Reply:
It is done to resolve pylint error R1718: (consider-using-set-comprehension)

https://python-reference.readthedocs.io/en/latest/docs/comprehensions/set_comprehension.html